packages/fieldset/test/lion-fieldset.test.js

```diff
@@ -614,24 +614,20 @@ describe('<lion-fieldset>', () => {
       });
     });
 
-    it('0 is a valid value to be serialized', async () => {
+    it('0 and empty string are valid values to be serialized', async () => {
+      // should just use `!= null` comparator to check for undefined and null.
       const fieldset = await fixture(html`
       <${tag}>
         <${childTag} name="price"></${childTag}>
+        <${childTag} name="firstName"></${childTag}>
       </${tag}>`);
       await nextFrame();
       fieldset.formElements.price.modelValue = 0;
-      expect(fieldset.serializeGroup()).to.deep.equal({ price: 0 });
-    });
-
-    it('__serializeElements serializes 0 as a valid value', async () => {
```

Comment:
Unnecessary test, and this method will probably be removed as it's only used by serializeGroup and not necessary for serializedValue